### DISCUSSION TITLE
Category / Subcategory  not working properly

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5734] [New Feature/Ideas Requests] [answered]
In order for categories to show subcategories, large category is activated on fluxbuilder and its depth is selected. When we do this, the category page also shows subcategories, but only the main categories appear on the archive page.

[https://streamable.com/fxnvvd](https://streamable.com/fxnvvd
)

**Comment by osman-kocak** [comment]
Like this 

[https://streamable.com/v6b1vd](https://streamable.com/v6b1vd)

**Comment by osman-kocak** [comment]
@minhcasi @luyx2412 @HanielQC @vumanhsn97

**Comment by HanielQC** [comment, marked as answer]
Dear Sir, 

Please post ticket on our support center for support.
This place is for the new feature request, so may I close this topic.

Thank you
